From 8b81855a04189920a2667a86a6bdea5860328059 Mon Sep 17 00:00:00 2001
From: Martin Lang <martin.lang@mpsd.mpg.de>
Date: Wed, 19 Nov 2025 11:30:38 +0100
Subject: [PATCH 1/7] Change name of binary from `sd` to `uppasd`

- The name `sd` conflicts with https://github.com/conda-forge/sd-feedstock
- Upstream will change the name to uppasd in the next release
---
 CMakeLists.txt | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c7620d3..b39a319 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -86,7 +86,7 @@ ENDIF(FCNAME STREQUAL "pgf90")
 ##############################################################################################
 # Define the executable name
 ##############################################################################################
-set(ASD_binary sd)
+set(ASD_binary uppasd)
 #if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
 #   string(APPEND CMAKE_Fortran_COMPILER ".exe")
 #   #message(STATUS "---> ${CMAKE_Fortran_COMPILER}")
@@ -94,13 +94,6 @@ set(ASD_binary sd)
 # Find the vernacular compiler name
 string(REGEX MATCH gfortran|g95|ifort|pgf90|f95|f77|nv current_compiler  ${CMAKE_Fortran_COMPILER})
 message(STATUS "Matched compiler: ${current_compiler}.  ${CMAKE_Fortran_COMPILER} ${FCNAME}")
-if(current_compiler)
-   string(APPEND ASD_binary ".${current_compiler}")
-endif()
-# add a `cuda` tag to the binary if used.
-if(USE_CUDA)
-   string(APPEND ASD_binary ".cuda")
-endif()
 SET(UppASD_EXE ${ASD_binary})
 message(STATUS "ASD_binary ${ASD_binary}.")
 ##############################################################################################
